README.md

```diff
@@ -6,54 +6,55 @@ A tool used to validate firefox experiments
 
-Here is an example: ```docker run --rm --name "klaatu" -v $PWD/{PATH-TO-XPI-FOLDER}:/code/test_files mozilla/klaatu:latest tox -e exp-tests -- --experiment=test_files/{NAME-OF-FILE}.xpi --html=report.html```
-
-## Included tests
-- ```test_experiment_does_not_stop_startup```: Experiment does not stop browser startup, or prohibit a clean exit.
-- ```test_private_browsing_disables_experiment```: Experiment should be disabled in private browsing mode.
-- ```test_experiment_does_not_drastically_slow_firefox```: Experiment should not slow firefox down by more then 20%.
-- ```test_experiment_shows_on_support_page```: Experiment should show on about:support page.
-- ```test_experiment_shows_on_studies_page```: Experiment should show on about:studies page.
-- ```test_experiment_expires_correctly```: Experiment should not be included in the sent pings when it is disabled/expired.
-- ```test_experiment_remains_disabled_after_user_disables_it```: Disable experiment, restart Firefox to make sure it stays disabled.
-- ```test_experiment_sends_correct_telemetry```: Make sure telemetry is sent and recieved properly.
-- ```test_experiment_does_not_stop_update```: Experiment should not block firefox updates.
+Here is an example: ```docker compose run klaatu tox -e bdd-tests -- --experiment-branch control --variables tests/fixtures/test_experiment.json --private-browsing-enabled```. By default this will build and run the tests on Firefox Release.
 
+There are 3 versions of Klaatu that have each major Firefox type. 
+1. `mozilla/klaatu:firefox-release` : Firefox Release
+2. `mozilla/klaatu:firefox-beta`: Firefox Beta
+3. `mozilla/klaatu:firefox-nightly`: Firefox nightly
 
 ## Prerequisites
 
 You should have docker and git installed.
 
-## How to use
+## Building locally
 
 1. Clone repository
-2. Fill out this JSON file, name it `variables.json` and place it in the root directory:
-
+2. Copy the JSON of your experiment, into a file named `experiment.json`. The JSON can be found in 2 ways. On production:
+```
+https://experimenter.services.mozilla.com/api/v6/experiments/{EXPERIMENT-SLUG-HERE}
 ```
-    {
-        "recipeId":
-        "slug":
-        "userFacingName":
-        "userFacingDescription":
-        "branch":
-        "active": true,
-        "addonId":
-        "addonUrl": "https://example.com/{YOUR ADDON ID HERE}@mozilla.org-signed.xpi",
-        "addonVersion":
-        "extensionApiId":
-        "extensionHash": "badhash",
-        "hashAlgorithm": "sha256",
-        "studyEndDate": null
-    }
+Or Stage:
 ```
-Add the path using the ```--variables``` option. ```--variables={PATH/TO/variables.json}```
+https://stage.experimenter.nonprod.dataops.mozgcp.net/api/v6/experiments/{EXPERIMENT-SLUG-HERE}
+```
+Place this file in the somewhere within the working directory.
+
+3. Add the path using the ```--variables``` option. Ex: ```--variables={PATH/TO/experiment.json}```
```

Comment:
Do we need to show an example of variables.json? Looks like we deleted it in the diff here.

Reply:
Nah, it's just JSON